### ISSUE TITLE
No results when running on Alfred 3

### ISSUE THREAD

**Issue by matthewmorek** [willfarrell/alfred-caniuse-workflow#13]
After migrating to Alfred 3, then updating the workflow to `v1.7`, I have noticed that no results are produced for any query searched for with `caniuse` keyword.

<img width="607" alt="screen shot 2016-09-15 at 11 06 45" src="https://cloud.githubusercontent.com/assets/1137994/18546193/83106ee8-7b34-11e6-9ce8-9aa6e258f5a1.png">

After running a debug within Alfred, I have stumbled upon this in the debug log:
```shell
[2016-09-15 11:06:43][STDERR: input.scriptfilter] mkdir: /Users/{user}/Library/Caches/com.runningwithcrayons.Alfred-2/Workflow Data: No such file or directory
mkdir: /Users/{user}/Library/Application Support/Alfred 2/Workflow Data: No such file or directory

[2016-09-15 11:06:43][ERROR: input.scriptfilter] JSON error: JSON text did not start with array or object and option to allow fragments not set. in JSON:
Warning: file_get_contents(): SSL operation failed with code 1. OpenSSL Error messages:
error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed in /Users/{user}/Dropbox/Important/Alfred/Alfred.alfredpreferences/workflows/user.workflow.172B379D-3BC8-4EB8-89C7-D746A95C8087/caniuse.php on line 31

Warning: file_get_contents(): Failed to enable crypto in /Users/{user}/Dropbox/Important/Alfred/Alfred.alfredpreferences/workflows/user.workflow.172B379D-3BC8-4EB8-89C7-D746A95C8087/caniuse.php on line 31

Warning: Invalid argument supplied for foreach() in /Users/{user}/Dropbox/Important/Alfred/Alfred.alfredpreferences/workflows/user.workflow.172B379D-3BC8-4EB8-89C7-D746A95C8087/caniuse.php on line 33
```
Things I did:
- [x] Update/re-install `caniuse` workflow
- [x] Quit/re-launch Alfred
- [x] Clear app's local storage cache, then re-launch Alfred
- [x] Give up ☹️

The script should use the correct Alfred 3 storage/cache locations, as far as I'm aware.
```
/Users/mmorek/Library/Caches/com.runningwithcrayons.Alfred-3
/Users/mmorek/Library/Application Support/Alfred 3/Workflow Data

**Comment by iagomelanias** [comment]
Same here 👍

**Comment by willfarrell** [comment]
Thanks for letting me know. Git hub updated their SSL, causing an issue with the php function. Fixed in v1.8.

**Comment by iagomelanias** [comment]
Thanks!
